.github/workflows/test-e2e.yml

```diff
@@ -51,21 +52,68 @@ jobs:
       - id: prepare-detox-build
         run: RN_SRC_EXT=e2e.ts yarn detox build -c ios.sim.release
         shell: bash
+      - id: upload-detox-build
+        uses: actions/upload-artifact@26f96dfa697d77e81fd5907df203aa23a56210a8 # v4.0.3
+        with:
+          name: ItaliaApp.app
+          path: ios/build/Build/Products/Release-iphonesimulator/ItaliaApp.app
+          retention-days: 2
+  run-e2e-tests:    
+    needs: build-detox-app
+    runs-on: macos-latest
+    environment: dev
+    strategy: 
+      fail-fast: false
+      matrix:
+        test: 
+          - "ts/__e2e__/"
+          - "ts/features/bonus/cgn/__e2e__/"
+          - "ts/features/messages/__e2e__/"
+          - "ts/features/euCovidCert/__e2e__/"
+          - "ts/features/wallet/onboarding/__e2e__/"
+    steps:
+      - id: install-applesimutils
+        run: |
+          brew tap wix/brew
+          brew install applesimutils
+        shell: bash
+      - id: checkout
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+      - id: download-detox-build
+        uses: actions/download-artifact@6b208ae046db98c579e8a3aa621ab581ff575935 # v4.1.1
+        with:
+          name: ItaliaApp.app
+          path: ios/build/Build/Products/Release-iphonesimulator/ItaliaApp.app
+      - id: list-app
+        run: |
+          pwd
+          ls -al ios/build/Build/Products/Release-iphonesimulator/
+        shell: bash
+      - id: setup
+        uses: ./.github/actions/setup-composite
+        with:
+          use-cache: 'true'
       - id: checkout-dev-server
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
         with:
           repository: pagopa/io-dev-api-server
           path: './_io-dev-api-server_'
       - id: run-e2e-tests
-        run: bash ./.github/scripts/run-e2e-tests.sh
+        run: bash ./.github/scripts/run-e2e-tests.sh ${{ matrix.test }}
       - id: notify-test-failure
         if: failure()
         uses: ./.github/actions/notify-e2e
         env:
+          TEST: ${{ matrix.test }}
```

Comment:
```suggestion
```
We don't use this environment variable. Am I right?

Reply:
I just added the variable in the e2e notifier in the python script that sends the slack message here to add the related test which is failing in the text message (can see it [here](https://github.com/pagopa/io-app/pull/5616/files#diff-222f6019b73edea2ebfb19d5fbcb2d5eafc7aceb642184a2abf6c338da8acf01R13))

Reply:
Okay! Perfect then!